src/Storages/Streaming/StorageRandom.h

```diff
@@ -13,7 +14,8 @@ namespace DB
 class ASTStorage;
 
 #define STORAGE_RANDOM_RELATED_SETTINGS(M) \
-    M(UInt64, eps, 1000, "Limits how many rows of data can be generated per window by random storages total streams", 0)
+    M(UInt64, eps, 1000, "Limits how many rows of data can be generated per second by random storages per stream (each query will generate $num_streams source by default). 0 means no maximum value", 0) \
```

Comment:
`Limit how many rows to be generated per second for each thread. Used by RANDOM STREAM. 0 means no limit.`